src/promptflow/promptflow/_core/connection_manager.py

```diff
@@ -44,6 +45,12 @@ def _build_connections(cls, _dict: Dict[str, dict]):
         for key, connection_dict in _dict.items():
             if isinstance(connection_dict, _Connection):
```

Comment:
@wangchao1230 another thing to discuss, can we put the connection object's handling logic outside of executor? Offline synced with @brynn-code , currently ConnectionManager is designed only support dict type connections.

Reply:
The question related to this is: do we need to support token-based connection object by allow it pass to executor now?

Reply:
Offline synced, will change to dict